packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/index.tsx

```diff
@@ -39,8 +40,13 @@ class Checkout extends PureComponent<Props> {
       this.props.simpleBuyActions.fetchSDDEligible()
       this.props.simpleBuyActions.fetchSBCards()
       this.props.brokerageActions.fetchBankTransferAccounts()
-      this.props.simpleBuyActions.fetchLimits(this.props.fiatCurrency || 'USD')
     }
+    // we need to re-fetch limits for SB
```

Comment:
please explain "why" in the comment, we already know the "what" :)

Reply:
oh yeah, because we call it here https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/3101/files#diff-7eb86189b3d4261d7094ff70289228c91e6b3c3a976934f2e86254280aeff3f6L27 without additional parameters, will add it in comment ;)

Reply:
thank you

Reply:
@schnogz added ;)